.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 node_modules
+dist
```

Comment:
`dist` is meant to be shipped.
If you want to ignore it, you'll need to change the build process, otherwise it won't be there when you uinstall it as a dependency